soc/nxp/imxrt/imxrt6xx/Kconfig.defconfig

```diff
@@ -115,3 +115,32 @@ choice USB_MCUX_CONTROLLER_TYPE
 endchoice
 
 endif # SOC_MIMXRT685S_CM33
+
+if SOC_MIMXRT685S_ADSP
+
+config SOC_TOOLCHAIN_NAME
+	string
+	default "nxp_rt600_adsp"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 198000000
+
+config XTENSA_CCOUNT_HZ
+	default SYS_CLOCK_HW_CYCLES_PER_SEC
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 1000
+
+config CACHE
+	default n
+
+config DCACHE
+	default n
+
+config CACHE_MANAGEMENT
+	default n
+
+config LOG
```

Comment:
Same here. Isn't this disabled by default?

Reply:
Yes. Thi sis defined like this:

```

config LOG                                                                                                                                                                                                         
»       bool "Logging"
»       select PRINTK if USERSPACE
»       help
»         Global switch for the logger, when turned off log calls will not be
»         compiled in.
```

and looks like if default value is not present than is `N`.